### DISCUSSION TITLE
network/RSS

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#194] [General]
# network/RSS

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/network/RSS.html

**Comment by BillXiang** [comment]
“RSS 利用网卡多队列特性，将每个核分别跟网卡的一个首发队列绑定”, ”首发队列“应该是“收发队列“

**Reply by isno** [reply]
谢谢，是我写错了，已修改。

**Comment by echo1937** [comment]
所以他们一个是队列数量, 一个是队列大小?

**Reply by isno** [reply]
是的，
